staging/src/k8s.io/client-go/tools/cache/fifo.go

```diff
@@ -145,7 +145,7 @@ func (f *FIFO) Close() {
 }
 
 // HasSynced returns true if an Add/Update/Delete/AddIfNotPresent are called first,
-// or an Update called first but the first batch of items inserted by Replace() has been popped
+// or the first batch of items inserted by Replace() has been popped.
```

Comment:
As long as the first `Replace()` is called first, it doesn't really matter whether `Add/Update/Delete` are called after. All it maters is whether the first batch of items are popped. Therefore I made the comments shorter and easier to follow.